api/task.go

```diff
@@ -171,7 +171,7 @@ func (h TaskHandler) Delete(ctx *gin.Context) {
 			return
 		}
 	}
-	result = h.DB.Select(clause.Associations).Delete(task)
```

Comment:
Rely on cascade delete.
This can be dangerous because it will delete everything associated with the task may not always correct.